### PR TITLE
Fix: Add link for Finite Element Families

### PR DESCRIPTION
**Pull request by qwinters**
Addresses FIXME comment in current documentation

(Not sure if this is the only file that needs to be edited for this change, but at least from looking at the website it _seemed_ like this was the right spot to edit)

### CODE REVIEW

**Comment by jorgensd** [comment]
This is not 100 % correct, as UFL supports elements we do not support in DOLFINx. 
The closest to a complete list of elements would https://docs.fenicsproject.org/basix/main/.
@mscroggs  Are there any elements listed in the basix link above we do not support in FFCx/DOLFINx?

**Comment by mscroggs** [comment]
Hermite are supported in Basix, but do not yet work in FFCx. Otherwise all elements on the Basix list are supported in DOLFINx. (The only other elements DOLFINx supports above these are QuadratureElements I think.)

**Comment by mscroggs** [comment]
potentially useful link: https://defelement.com/lists/implementations/basix.html

File: chapter1/fundamentals_code.ipynb [hunk 1]
```diff
@@ -125,7 +125,7 @@
     "DOLFINx supports a large variety on elements on simplices \n",
     "(triangles and tetrahedra) and non-simplices (quadrilaterals\n",
     "and hexahedra). For an overview, see:\n",
-    "*FIXME: Add link to all the elements we support*\n",
+    "[basix's element families](https://docs.fenicsproject.org/basix/main/python/_autosummary/basix.html#basix.ElementFamily) _Note that Hermite elements are not currently supported._\n",
```
Comment (jorgensd) [inline chapter1/fundamentals_code.ipynb:128]:
```suggestion
    "[Basix's element families](https://docs.fenicsproject.org/basix/main/python/_autosummary/basix.html#basix.ElementFamily) _Note that Hermite elements are not currently supported._\n",
```

**Comment by jorgensd** [comment]
@qwinters You need to also update the `.py` file. This should be done automatically if you have `jupytext` installed on your system when launching the notebook.

**Pull request closed without merging**